testsuite/forge-cli/src/suites/realistic_environment.rs

```diff
@@ -323,9 +324,7 @@ pub(crate) fn realistic_env_max_load_test(
         success_criteria = success_criteria.add_latency_breakdown_threshold(
             LatencyBreakdownThreshold::new_with_breach_pct(
                 vec![
-                    (LatencyBreakdownSlice::QsBatchToPos, 0.35),
-                    // quorum store backpressure is relaxed, so queueing happens here
-                    (LatencyBreakdownSlice::QsPosToProposal, 2.5),
+                    (LatencyBreakdownSlice::MempoolToBlockCreation, 0.35 + 2.5),
```

Comment:
add back the comment for
// quorum store backpressure is relaxed, so queueing happens here

to explain the high limit

Reply:
Done!